pkg/registry/apps/replicaset/storage/storage_test.go

```diff
@@ -158,6 +158,9 @@ func TestGenerationNumber(t *testing.T) {
 	modifiedSno.Status.ObservedGeneration = 10
 	ctx := genericapirequest.NewDefaultContext()
 	rs, err := createReplicaSet(storage.ReplicaSet, modifiedSno, t)
+	if err != nil {
+		t.Errorf("unexpected error: %v", err)
```

Comment:
nice catch! :+1:

Reply:
can you fix this for replicationcontroller as well?

https://github.com/kubernetes/kubernetes/blob/d3bed1e5c99511b60513b0812cafc9244d4b0b56/pkg/registry/core/replicationcontroller/storage/storage_test.go#L163

Reply:
Done